elmslie-core/src/main/java/vivid/money/elmslie/core/store/ElmStore.kt

```diff
@@ -22,8 +22,9 @@ class ElmStore<Event : Any, State : Any, Effect : Any, Command : Any>(
     initialState: State,
     private val reducer: StateReducer<Event, State, Effect, Command>,
     private val actor: Actor<Command, out Event>,
+    storeListeners: Set<StoreListener<Event, State, Effect, Command>>? = null,
```

Comment:
I think would be better to make it private at all..and instead of attach/detach, do add/remove to the collection..